sale_order_line_description/__init__.py

```diff
@@ -1 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
License is not necessary on init files

Reply:
I know it's not necessary but I saw many module in OCA have license on init files and I think this module is open source that why I add License to init files.